oeps/oep-2.rst

```diff
@@ -0,0 +1,101 @@
+===================================
+OEP-2: Accordance to Best Practices
+===================================
+
++---------------+-------------------------------------------+
+| OEP           | :doc:`oep-2`                              |
++---------------+-------------------------------------------+
+| Title         | Accordance to Best Practices              |
++---------------+-------------------------------------------+
+| Last-Modified | 2016-04-21                                |
++---------------+-------------------------------------------+
+| Author        | Calen Pennington <cale@edx.org>           |
++---------------+-------------------------------------------+
+| Arbiter       |                                           |
++---------------+-------------------------------------------+
+| Status        | Draft                                     |
++---------------+-------------------------------------------+
+| Type          | Best Practice                             |
++---------------+-------------------------------------------+
+| Created       | 2016-04-21                                |
++---------------+-------------------------------------------+
+
+Abstract
+========
+
+Proposes a standard format for repository metadata to self-report on
+accordance to all Best Practice OEPs.
```

Comment:
This document already covers more than just OEPs, so maybe we shouldn't focus on that.  Conformance to best practice OEPs is just one kind of information in this data.